Core1Platform/01_General.md

```diff
@@ -0,0 +1,6 @@
+# General information
+
+The *Actindo Core1 Platform* offers a comprehensive solution for efficient digital commerce. It is an API and cloud-based *Digital Operations Platform (DOP)* that enables to centrally manage all key processes and touchpoints to create a better customer experience. It brings together automation and workflows, and offers tremendous relief and time savings in day-to-day operations. From product data, purchasing and inventory to customer management, monitoring and reporting, everything is made available centrally in one place.  
```

Comment:
call it API-first and cloud-native to stay within the MACH-Definition

<img width="690" alt="image" src="https://github.com/Actindo-AG/Documentation/assets/25777820/6fcc830b-8cdb-4a3d-b490-5f0ff1e48e7d">

I think to be further within the MACH scope and emphasize the "M" inside mach one should add something like
"The DOP is composed of multiple independent services and can act as a standalone solution as well as only providing the needed components for an existing as well as newly created IT landscape"

Something like this. I know that it is mentioned later on in the next chapter but important to have this information right at the beginning